src/api/procedures/manageAllowedVenues.ts

```diff
@@ -0,0 +1,53 @@
+import { Asset, Procedure } from '~/internal';
+import { ManageAllowedVenuesParams, TxTags } from '~/types';
+import { ExtrinsicParams, ProcedureAuthorization, TransactionSpec } from '~/types/internal';
+
+/**
+ * @hidden
+ */
+export type Params = ManageAllowedVenuesParams;
+
+/**
+ * @hidden
+ */
+export async function prepareManageAllowedVenues(
+  this: Procedure<Params, void>,
+  args: Params
+): Promise<TransactionSpec<void, ExtrinsicParams<'settlement', 'allowVenues'>>> {
+  const {
+    context: {
+      polymeshApi: { tx },
+    },
+  } = this;
+
+  const { ticker, venues, action } = args;
+
+  return {
+    transaction: action === 'allow' ? tx.settlement.allowVenues : tx.settlement.disallowVenues,
+    args: [ticker, venues],
```

Comment:
We do not directly add in the ticker as params, we first need to serialize it in a way that chain won't complain. SDK can accept plain string as ticker values, but if it is < 12 characters the chain will complain.

Use `tickerToString(ticker)` to get the rawTicker value that should be sent.

Reply:
Similarly, convert the venues using `bigNumberToU64` and then send in the values